app/src/main/java/com/example/bfinerocks/flashcard/adapters/DeckListCustomAdapter.java

```diff
@@ -0,0 +1,56 @@
+package com.example.bfinerocks.flashcard.adapters;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.TextView;
+
+import com.example.bfinerocks.flashcard.R;
+import com.example.bfinerocks.flashcard.models.Deck;
+
+import java.util.List;
+
+/**
+ * Created by BFineRocks on 12/30/14.
+ */
+public class DeckListCustomAdapter extends ArrayAdapter<Deck> {
+
+    private static class ViewHolder{
+        TextView deckName;
+    }
+
+    private int layoutResource;
+
+    public DeckListCustomAdapter(Context context, int resource, List<Deck> objects) {
+        super(context, resource, objects);
+        this.layoutResource = resource;
+    }
+
+    @Override
+    public View getView(int position, View convertView, ViewGroup parent) {
+        Log.i("viewCalled", "view Called");
```

Comment:
This looks like it was for debugging. Probably remove it once you've confirmed whatever it was for.